docs/reference/sso.md

```diff
@@ -0,0 +1,511 @@
+---
+layout: default
+title: Single Sign On (SSO)
+description: This section covers authentication (using SSO) of your lakeFS Cloud or lakeFS Enterprise.
+parent: Reference
+nav_order: 65
+has_children: false
+---
+
+
+# Single Sign On (SSO)
+{: .d-inline-block }
+Cloud
+{: .label .label-green }
+
+Enterprise
+{: .label .label-purple }
+{: .no_toc }
+
+
+{: .note}
+> SSO is only available for [lakeFS Cloud](../cloud.md) and [lakeFS Enterprise](../enterprise.md).
+>
+> Using the Open Source? Read more on [authentication](authentication.html). 
+
+{% include toc.html %}
+
+## Supported Protocols and Third Parties
+
+lakeFS Cloud and lakeFS Enterprise support the commonly used protocols for authentication, such as OpenID Connect and SAML.
+
+There are specific authentication utilities that we've already verified that are working, if you're using one that is not listed below but is based on OpenID Connect or SAML, it's probably a small effort for us to make sure it'll work, please [contact us](support@treeverse.io) for more information.
+
+## Cloud
+
+lakeFS Cloud leverage Auth0's capabilities for authentication, therefore, we're supporting Auth0's identity providers, such as: Active Directory/LDAP, ADFS, Azure Active Directory Native, Google Workspace, OpenID Connect, Okta, PingFederate, SAML and Azure Active Directory.
+
+### Okta
+{: .d-inline-block }
+Cloud
+{: .label .label-green }
+
+Steps:
+1. Login to your Okta account
+2. Select **Applications > Applications**, then **Create App Integration**.
+3. Select Create New App and enter the following:
+    1. For **Sign-in method**, choose OIDC.
+    2. Under **Application type**, choose **Web app**.
+    3. Select Next.
+4. Under General Settings:
+    1. **App integration name**, enter a name for your application. (i.e lakeFS Cloud)
+5. In the **Sign-in redirect URIs** field, enter https://lakefs-cloud.us.auth0.com/login (United States) or https://lakefs-cloud.eu.auth0.com/login (Europe).
+6. Under **Sign-in redirect URIs**, click **Add URI**, enter https://lakefs-cloud.us.auth0.com/login/callback (United States) or https://lakefs-cloud.eu.auth0.com/login/callback (Europe).
+7. Under **Assignments**, choose the wanted **Controlled access**. (i.e Allow everyone in your organization to access)
+8. Uncheck **Enable immediate access with Federation Broker Mode**.
+9. Select **Save**.
```

Comment:
Mostly because we have application specific settings in our steps (e.g the redirect URI, the "Enable immediate access with Federation Broker Mode" checkbox).
These steps focus on exactly what's needed for lakeFS.cloud compared to [Okta's guide](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_OIDC.htm) which covers much more steps and configuration options.
I think that maybe adding a reference to that guide for whoever who wants to use the full guide, might help, I'm not sure about replacing it altogether.
WDYT?